### ISSUE TITLE
loss dramatically increases and the training process is ruined, why?

### ISSUE THREAD

**Issue by machengcheng2016** [soo89/ISD-SSD#3]
Greetings!
I am trying to run `train_ssd.py` and `train_isd.py`.
However, I am always meeting the situation that loss increases dramatically during the training process. The loss is going like
`Loss: 14.6995.
 Loss: 12.2860.
Loss: 14.0080.
Loss: 20.0848.
Loss: 29821698048.0000 `
Then the training process breaks. 
This situation usually happen at the first 1k iterations, which is really weird.  Do you know why?

I notice that there is a `train_isd.sh` script in this repo. Besides, you add a `if break` sequence in `train_ssd.py` and `train_isd.py`. Seems you also met this situation huh? 
You might say this problem comes from the original [ssd.pytorch](https://github.com/amdegroot/ssd.pytorch) and suggest me decreasing the learning rate, since the model fails to converge. But when I try to run the training script a few more times just as your `train_isd.sh` does, this problem can sometimes disappear. Besides, changing the learning rate will bring more trouble. So once again, why will this problem happen, and how can I fix it?

Thank you!

**Comment by soo89** [comment]
We met the problem what you happened. This occurs in the initial process of training.
And, we tried to re-train if the loss becomes too large.
(Check our train_isd.py L328)

**Comment by machengcheng2016** [comment]
Yes. That is what I'm talking about. You add a if break sequence in `train_ssd.py` and `train_isd.py`.
Please check my other [issue](https://github.com/soo89/ISD-SSD/issues/4)
